pages/api/download-subdirectory.ts

```diff
@@ -0,0 +1,30 @@
+import { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
why do we have this dup file?

Seems like the only diff is camel case and res status:

![Screen Shot 2021-07-26 at 10 00 45 AM](https://user-images.githubusercontent.com/31771087/127001618-5d9ed657-3b02-48aa-8a5a-5d790d3155e0.png)

Reply:
I had to change my HEAD in git so this must be a fragment of that. whoops